### PR TITLE
Preset/dashboard-borders

### PR DESCRIPTION
**Pull request by gadenbuie**
* Restores the card border for `.card-header` in the shiny preset dashboard styles.

* Softens the general transparent border for both light and dark mode.

I added a screenshot in the commit history so you can see the [before/after](https://github.com/rstudio/bslib/pull/893/files/a931f0d87b3ac929c3eb43d79ef7319203abdc44..b6d9baad52778c3ec901f51d13bd267ff129bfa7#diff-5d602c03e2f1431c7b7d9f6d7c302b348664d5f944aba6d834d9db3d63de2ad5).

**TODO**

- [x] Remove the screenshot

### CODE REVIEW

File: inst/builtin/bs5/shiny/_rules.scss [hunk 1]
```diff
@@ -303,6 +308,10 @@ $icon-classes: ("bi", "fa", "fas", "far", "fab", "material-icons") !default;
 }
 
 @include color-mode(dark) {
+  .card {
+    --card-border-color: rgba(255, 255, 255, 0.1);
+  }
+
```
Comment (gadenbuie) [inline inst/builtin/bs5/shiny/_rules.scss:314]: Oops, this isn't needed. I added this before I found `$border-color-translucent-dark`

```suggestion
```

File: inst/builtin/bs5/shiny/_variables.scss [hunk 2]
```diff
@@ -108,8 +108,13 @@ $box-shadow-dark:
   0px 3px 7px rgba(0, 0, 0,.1),
   0px 12px 30px rgba(0, 0, 0,.08) !default;
 
+// Cards
 $card-border-radius: 8px !default;
 
+// Borders
+$border-color-translucent: RGBA(0, 0, 0, 0.1) !default;
```
Comment (gregswinehart) [inline inst/builtin/bs5/shiny/_variables.scss:115]: If you're open to a more detailed $gray-200 comp in RGBA, I think it'd be `RGBA(40, 70, 94, 0.1)`:

colors

**Review by gregswinehart — CHANGES_REQUESTED** [review]
This looks great, just one suggestion in the RGBA if you're up for it.

File: inst/builtin/bs5/shiny/_variables.scss (hunk 2 above)
Comment (gadenbuie) [reply inst/builtin/bs5/shiny/_variables.scss:115]: I'm open to that! We'd need a similar color for dark mode, too

File: inst/builtin/bs5/shiny/_rules.scss [hunk 3]
```diff
@@ -75,12 +75,18 @@ $bslib-checkbox-radio-margin-right: 0.35em !default;
...
+    // Shadows on top-level cards/sidebars, not in nested components
     box-shadow: $box-shadow;
-    --#{$prefix}card-border-width: 0;
     .bslib-card, .bslib-sidebar-layout {
       box-shadow: none;
+    }
+
+    // Remove outer border, but keep inner borders and borders on nested components
+    --#{$prefix}card-border-width: 0;
+    .card-body, .card-header, .card-footer, .bslib-card, .bslib-sidebar-layout {
```
Comment (cpsievert) [inline inst/builtin/bs5/shiny/_rules.scss:86]: Any reason why this can't be generic?

```suggestion
    * {
```

Comment (cpsievert) [reply inst/builtin/bs5/shiny/_rules.scss:86]: Or maybe it would make sense to just do `border-width: 0;`?

Comment (gadenbuie) [reply inst/builtin/bs5/shiny/_rules.scss:86]: Yeah let's do this second option

**Comment by gadenbuie** [comment]
Preview after resolving the above two comments

image

**Review by cpsievert — APPROVED** [review]
Nice, thanks!

**Pull request merged**